### PR TITLE
343850043: Added utility scripts to plot tasks profiling data

### PR DESCRIPTION
**Pull request by joachimmetz**
[Code review: 343850043: Added utility scripts to plot tasks profiling data](https://codereview.appspot.com/343850043/)

### CODE REVIEW

File: utils/plot-tasks.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
```
Comment (Onager) [inline utils/plot-tasks.py]: Please rename to plot_tasks, and remove this pylint override:
https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming 
https://google.github.io/styleguide/pyguide.html?showone=Main#Main
"Even a file meant to be used as a script should be importable"

Yes, we have other files that aren't compliant at present, but we should fix those up.

File: utils/plot-tasks.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+# mathplotlib does not support Unicode strings a column names.
+# from __future__ import unicode_literals
+
+import argparse
+import glob
+import os
+import sys
+
+from matplotlib import pyplot  # pylint: disable=import-error
+from numpy import genfromtxt  # pylint: disable=import-error
```
Comment (Onager) [inline utils/plot-tasks.py]: genfromtxt is a function, import the module or package instead: https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

File: utils/plot-tasks.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+"""Script to plot tasks from profiling data.
+
+This script requires the matplotlib and numpy Python modules.
+"""
+
+from __future__ import print_function
+# mathplotlib does not support Unicode strings a column names.
```
Comment (Onager) [inline utils/plot-tasks.py]: Isn't this just a problem when setting the column names in pyplot then? Using byte strings here is going to cause compatibility problems in the near future. Please reconsider this.

File: utils/plot-tasks.py [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+    print('No such directory: {0:s}'.format(options.profile_path))
+    return False
+
+  names = ['time', 'identifier', 'status']
+
+  measurements = {}
+
+  glob_expression = os.path.join(options.profile_path, 'tasks-*.csv.gz')
+  for csv_file in glob.glob(glob_expression):
+    data = genfromtxt(
```
Comment (Onager) [inline utils/plot-tasks.py]: Please set the encoding here as well

File: utils/plot-tasks.py [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+  if not os.path.isdir(options.profile_path):
+    print('No such directory: {0:s}'.format(options.profile_path))
+    return False
+
+  names = ['time', 'identifier', 'status']
+
+  measurements = {}
+
+  glob_expression = os.path.join(options.profile_path, 'tasks-*.csv.gz')
+  for csv_file in glob.glob(glob_expression):
```
Comment (Onager) [inline utils/plot-tasks.py]: rename to csv_file_name

File: utils/plot-tasks.py (hunk 1 above)
Comment (joachimmetz) [reply utils/plot-tasks.py]: done

File: utils/plot-tasks.py (hunk 2 above)
Comment (joachimmetz) [reply utils/plot-tasks.py]: ack

File: utils/plot-tasks.py (hunk 5 above)
Comment (joachimmetz) [reply utils/plot-tasks.py]: done

File: utils/plot-tasks.py (hunk 4 above)
Comment (joachimmetz) [reply utils/plot-tasks.py]: done

File: utils/plot-tasks.py (hunk 3 above)
Comment (joachimmetz) [reply utils/plot-tasks.py]: if I recall correctly, bytes column names cause issues for python3 and unicode string for python2. For now it is not problematic if these scripts only support 1 python version.

**Pull request merged**